src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -34,6 +34,10 @@ class ThirdPartyAIService extends Component {
   };
 
   fetchServiceSpec = async (org_id, service_id) => {
+    if(process.env.REACT_APP_SANDBOX) {
```

Comment:
process.env variables would always be casted to Strings.. so this `might` be always true.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309243540)